OSDOCS-13791#adding RN text (#91810)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

[id="ocp-release-notes-installation-and-update_{context}"]
=== Installation and update
[id="ocp-4-19-installation-and-update-remove-terraform-ibm-cloud_{context}"]
==== Cluster API replaces Terraform for {ibm-cloud-title} installations

In this release of {product-title}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.

> For more information, see...

Reviewer: need not it

[id="ocp-release-notes-installation-and-update_{context}"]
=== Installation and update

[id="ocp-4-19-installation-and-update-remove-terraform-ibm-cloud_{context}"]
==== Cluster API replaces Terraform for {ibm-cloud-title} installations

> In {product-title} {ocp-nplus1}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.

Reviewer 2: It seems, from a quick search in Atom, that _the Cluster API_ is far more common. 

```suggestion
In {product-title} {ocp-nplus1}, the installation program uses the Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
```

Reviewer 2: I see `{ocp-nplus1}` used in the docs only once. It is [intended to be used to specify the next OCP version](https://github.com/openshift/openshift-docs/pull/86479), not the current. And, I think it will get updated each version. 
```suggestion
In {product-title} {product-version}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
```

Reviewer 2: It seems, from a quick search in Atom, that _the Cluster API_ is far more common. 

```suggestion
In {product-title} {ocp-nplus1}, the installation program uses the Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
```
